osu.Game/GameModes/Play/Player.cs

```diff
@@ -24,12 +29,48 @@ public class Player : OsuGameMode
 
         public BeatmapInfo BeatmapInfo;
 
+        PlayerInputManager inputManager;
+
+        class PlayerInputManager : UserInputManager
+        {
+
+            protected override Quad InputManagerMappingQuad => Host.ScreenSpaceDrawQuad;
+
+            public PlayerInputManager(BasicGameHost host)
+                : base(host)
+            {
+            }
+
+            protected override void UpdateMouseState(InputState state)
+            {
+                base.UpdateMouseState(state);
+
+                MouseState mouse = (MouseState)state.Mouse;
+
+                foreach (Key k in state.Keyboard.Keys)
+                {
+                    switch (k)
+                    {
+                        case Key.Z:
+                            mouse.ButtonStates.Find(s => s.Button == MouseButton.Left).State = true;
+                            break;
+                        case Key.X:
+                            mouse.ButtonStates.Find(s => s.Button == MouseButton.Right).State = true;
+                            break;
+                    }
+                }

```

Comment:
I think it would be cleaner to have the things listening for mouse clicks also just check for the keypresses.